.babelrc

```diff
@@ -3,7 +3,6 @@
     [
       "env",
       {
-        "modules": false,
```

Comment:
Explanation: Jest expects `import` statements to be transpiled (see [jest issue](https://github.com/facebook/jest/issues/3202#issuecomment-290967270) for more context). I doubt this affects much but I wanted to point it out.

Instead of changing all envs, we could instead add a separate `test` env if needed.

Reply:
Does this mean if I try to use eg `import defaultExport from "module-name";` in some application code it won't work?

I worry this will lead to a really confusing bug down the road. Could you go ahead with the testing environment plan? Or if I've misunderstood this, help me out...

Reply:
ES6 imports should work with ES5 exports; this should behave as expected in this case. I'll add a `test` env just in case; we may as well have it for the future anyway.

Reply:
New envs are working.

I couldn't get either Jest or Webpacker to recognize `.babelrc.js` or `babel.config.js` (see relevent [babel docs](https://babeljs.io/docs/en/config-files#configuration-file-types) and [jest docs](https://jestjs.io/docs/en/23.x/getting-started.html#using-babel)).

That would have saved some duplication since it would support variables. I'm not sure why it isn't working because it's a common convention, but all it gives us is a more DRY babel config so it's probably not worth banging my head any longer.